packages/vm/tests/api/runBlockchain.spec.ts

```diff
@@ -70,11 +68,7 @@ tape('runBlockchain', (t) => {
 
     const vm = await setupVM({ common, genesisBlock })
 
-    st.equal(vm.blockchain.meta.genesis?.toString('hex'), testData.genesisBlockHeader.hash.slice(2))
-
     await vm.blockchain.putBlock(block)
-    const head = await vm.blockchain.getHead()
-    st.equal(head.hash().toString('hex'), testData.blocks[0].blockHeader.hash.slice(2))
```

Comment:
Same here I guess?

Reply:
while getHead() would have returned this block.  getIteratorHead() returns the genesis block if the iterator has not been run (with vm.runBlockchain())

Also does not work to move it after the runBlockchain() call because this is a test where runBlockchain() throws an error